octavia_f5/controller/worker/controller_worker.py

```diff
@@ -486,6 +486,9 @@ def ensure_amphora_exists(self, load_balancer_id):
                 load_balancer_id=load_balancer_id,
                 compute_flavor=CONF.host,
                 status=lib_consts.ACTIVE)
+            device_amp = self._amphora_repo.get(
+                db_apis.get_session(),
+                load_balancer_id=load_balancer_id)
```

Comment:
Instead, you can just return since we don't need to check the host of the amphora we just created.